lib/algorithms/aes.js

```diff
@@ -240,11 +244,18 @@ module.exports.AES_GCM = {
     const secretKey = key[kKeyMaterial];
     const cipher = `aes-${secretKey.symmetricKeySize << 3}-gcm`;
 
+    if (tagLength !== undefined && tagLength % 8 !== 0)
```

Comment:
I don't think you need the `undefined` check since `undefined % 8` is `NaN`.

Reply:
Maybe I'm missing something, but `NaN !== 0`, so `(undefined % 8 !== 0) === true`, but `undefined` is a valid input.

Reply:
Sorry, I read this incorrectly. You're correct.